packages/office-ui-fabric-react/src/components/Button/PrimaryButton/PrimaryButton.tsx

```diff
@@ -17,11 +16,10 @@ export class PrimaryButton extends BaseComponent<IButtonProps, {}> {
     const { theme, styles } = this.props;
 
     return (
-      <BaseButton
+      <DefaultButton
         { ...this.props }
-        variantClassName='ms-Button--primary'
```

Comment:
As you can see, the docs now line up better with the XD file

![image](https://user-images.githubusercontent.com/1434956/29626420-8386784c-87e3-11e7-8d67-bfe36b20e36f.png)